interfaces/thirdparty/cp2k.py

```diff
@@ -140,25 +142,82 @@ def get_runtime(self):
         diff = endTime - startTime
         return diff.total_seconds()
 
-    def _get_energy_type(self, search='Total', index=0):
+    def get_timings(self):
+        """Return the Timings from the End of the Output File"""
+        chunk = self.get_output_chunk(begin="SUBROUTINE",end="-"*70)
+        ret = {}
+        for line in chunk[1:]:
+            line = line.split()
+            ret[line[0]] = line[1:]
+        return ret
+
+
+    def _get_energy_type(self, search='Total', index=-1, unit='a.u.'):
         if index:
```

Comment:
WIll this behave as intended when `index=0`?
Considering `bool(0) is False`.

Reply:
True, fixed this. No need for this check anyhow.